src/Application/RegulationOrder/Command/CreateRegulationOrderCommand.php

```diff
@@ -11,5 +11,9 @@ final class CreateRegulationOrderCommand implements CommandInterface
     public string $description;
     public string $issuingAuthority;
     public ?\DateTimeInterface $startPeriod;
-    public ?\DateTimeInterface $endPeriod;
+    public ?\DateTimeInterface $endPeriod = null;
+    public ?float $maxWeight = null;
+    public ?float $maxHeight = null;
+    public ?float $maxWidth = null;
+    public ?float $maxLength = null;
```

Comment:
Remarque : si je ne mettais pas `= null`, et que je ne définissais pas `form["regulation_order[maxWeight]"]` dans les tests fonctionnels par ex, alors ceux-ci échouaient avec une erreur du type : "cannot use `$command->maxWeight` before it was assigned" dans l'exécution du handler.